brainstormforce/astra
# AST-2957 - WordPress 6.2 compatibility fixes

### Description
- gradients options needs to be pass to Gradientpicker component as this is required
- https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/gradient-picker/index.tsx

### Screenshots
- https://d.pr/i/y9URtf
- https://share.getcloudapp.com/NQuWJRdL

### Types of changes
- Bug fix

### How has this been tested?
- check the mega menu, it should load properly
- check the customizer's background control where the gradient component is being used

### Checklist:
